src/components/themed/EdgeText.tsx

```diff
@@ -82,15 +82,15 @@ export const EdgeText = (props: LabelProps) => {
   }
 
   return (
-    <Text
+    <UnscaledText
```

Comment:
Because `EdgeText` is used in a *ton* of places, the extra level of rendering will have a small but meaninful perf impact. Can we use the native `Text` element and then directly apply `allowFontScaling={false}` here, and here alone? This should minimize the perf hit without spreading too much hard-coded scaling logic over the codebase.